babel.config.js

```diff
@@ -1,15 +1,18 @@
-const { babelConfig: e2eBabelConfig } = require( '@woocommerce/e2e-environment' );
+const {
+	babelConfig: e2eBabelConfig,
+} = require( '@woocommerce/e2e-environment' );
 
-module.exports = function( api ) {
+module.exports = function ( api ) {
 	api.cache( true );
 
 	return {
-        ...e2eBabelConfig,
-        presets: [
-              ...e2eBabelConfig.presets,
-              '@wordpress/babel-preset-default',
-        ],
-        plugins: [
+		...e2eBabelConfig,
+		presets: [
+			...e2eBabelConfig.presets,
+			'@wordpress/babel-preset-default',
+		],
+		sourceType: 'unambiguous',
```

Comment:
This appears to be the only change in this file. Is it needed? Since we removed usage of `Array.prototype.flat` I'm wondering why change it because there appears to be a risk involved.

>unambiguous can be quite useful in contexts where the type is unknown, but it can lead to false matches because it's perfectly valid to have a module file that does not use import/export statements.

From [Babel docs](https://babeljs.io/docs/en/options#misc-options)

Reply:
It's needed to be able to compile the `common.js` file in the `debug` module.

Reply:
>It's needed to be able to compile the common.js file in the debug module.

Since this introduces some potential for problems, do you think we can apply this PR to the 1.8.x release branch and try to get CoreJS updated in main?

Reply:
@psealock Seems reasonable.